application/dependencies.json

```diff
@@ -1,260 +1,262 @@
-  "version": "2.0.5"
-}
-{
-  "name": "mlocati/ip-lib",
-  "version": "1.18.0"
-}
-{
-  "name": "monolog/monolog",
-  "version": "2.9.2"
-}
-{
-  "name": "notamedia/yii2-sentry",
-  "version": "1.7.0"
-}
-{
-  "name": "paragonie/constant_time_encoding",
-  "version": "v2.6.3"
-}
-{
-  "name": "paragonie/random_compat",
-  "version": "v9.99.100"
-}
-{
-  "name": "php-http/client-common",
-  "version": "2.7.0"
-}
-{
-  "name": "php-http/discovery",
-  "version": "1.19.1"
-}
-{
-  "name": "php-http/httplug",
-  "version": "2.4.0"
-}
-{
-  "name": "php-http/message",
-  "version": "1.16.0"
-}
-{
-  "name": "php-http/message-factory",
-  "version": "1.1.0"
-}
-{
-  "name": "php-http/promise",
-  "version": "1.2.0"
-}
-{
-  "name": "phpseclib/phpseclib",
-  "version": "3.0.33"
-}
-{
-  "name": "phpspec/php-diff",
-  "version": "v1.1.3"
-}
-{
-  "name": "psr/cache",
-  "version": "3.0.0"
-}
-{
-  "name": "psr/container",
-  "version": "2.0.2"
-}
-{
-  "name": "psr/http-client",
-  "version": "1.0.3"
-}
-{
-  "name": "psr/http-factory",
-  "version": "1.0.2"
-}
-{
-  "name": "psr/http-message",
-  "version": "1.1"
-}
-{
-  "name": "psr/log",
-  "version": "1.1.4"
-}
-{
-  "name": "ralouphie/getallheaders",
-  "version": "3.0.3"
-}
-{
-  "name": "roave/security-advisories",
-  "version": "dev-master d66cd3b"
-}
-{
-  "name": "sentry/sdk",
-  "version": "3.5.0"
-}
-{
-  "name": "sentry/sentry",
-  "version": "3.22.0"
-}
-{
-  "name": "silinternational/email-service-php-client",
-  "version": "2.4.1"
-}
-{
-  "name": "silinternational/idp-id-broker-php-client",
-  "version": "4.3.1"
-}
-{
-  "name": "silinternational/php-array-dot-notation",
-  "version": "0.1.0"
-}
-{
-  "name": "silinternational/php-env",
-  "version": "2.1.1"
-}
-{
-  "name": "silinternational/psr3-adapters",
-  "version": "3.1.0"
-}
-{
-  "name": "silinternational/yii2-json-log-targets",
-  "version": "2.1.0"
-}
-{
-  "name": "swiftmailer/swiftmailer",
-  "version": "v6.3.0"
-}
-{
-  "name": "symfony/deprecation-contracts",
-  "version": "v3.3.0"
-}
-{
-  "name": "symfony/http-client",
-  "version": "v6.3.7"
-}
-{
-  "name": "symfony/http-client-contracts",
-  "version": "v3.3.0"
-}
-{
-  "name": "symfony/options-resolver",
-  "version": "v6.3.0"
-}
-{
-  "name": "symfony/polyfill-iconv",
-  "version": "v1.28.0"
-}
-{
-  "name": "symfony/polyfill-intl-idn",
-  "version": "v1.28.0"
-}
-{
-  "name": "symfony/polyfill-intl-normalizer",
-  "version": "v1.28.0"
-}
-{
-  "name": "symfony/polyfill-mbstring",
-  "version": "v1.28.0"
-}
-{
-  "name": "symfony/polyfill-php72",
-  "version": "v1.28.0"
-}
-{
-  "name": "symfony/polyfill-php80",
-  "version": "v1.28.0"
-}
-{
-  "name": "symfony/service-contracts",
-  "version": "v3.3.0"
-}
-{
-  "name": "yiisoft/yii2",
-  "version": "2.0.49.2"
-}
-{
-  "name": "yiisoft/yii2-composer",
-  "version": "2.0.10"
-}
-{
-  "name": "yiisoft/yii2-gii",
-  "version": "2.2.6"
-}
-{
-  "name": "yiisoft/yii2-swiftmailer",
-  "version": "2.1.3"
-}
+[
```

Comment:
I thought JSON files were curly braced.

Reply:
The first character is only a curly-brace if the root level item in the JSON is an object. If it's an array, it's square braced. It's basically just "whatever is needed to encode the given value", which can be an array, an object, or even values like `true` or `"a string"`.